internal/resource/nvml-device.go

```diff
@@ -99,3 +101,21 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) GetFabricIDs() (string, string, error) {
+	gfInfo, ret := d.GetGpuFabricInfo()
+	if ret != nvml.SUCCESS {
+		return "", "", fmt.Errorf("failed to get GPU fabric info: %w", ret)
+	}
+
+	// ClusterUUID
```

Comment:
The comment is not required. The variable names already tell us what we're doing here.

Reply:
Done!